css/selectors/part.json

```diff
@@ -0,0 +1,62 @@
+{
+  "css": {
+    "selectors": {
+      "part": {
+        "__compat": {
+          "description": "<code>::part</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::part",
+          "support": {
+            "chrome": {
+              "version_added": "73"
+            },
+            "chrome_android": {
+              "version_added": "73"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
```

Comment:
We're actually in the process of removing Edge Mobile from compatibility tables.  Can we remove all Edge Mobile compatibility from these changes?